### PR TITLE
Add stream schema cli

### PR DESCRIPTION
**Pull request by sacloudy**
- add cli commands for stream schema related operations.
- to fix: run stream schema ginkgo tests together may fail.

### CODE REVIEW

**Comment by hanahmily** [comment]
@sacloudy Please fix issues raised by the checkers

File: bydbctl/internal/cmd/root.go [hunk 1]
```diff
@@ -30,7 +32,33 @@ func NewRoot() *cobra.Command {
...
+				if os.IsNotExist(err) {
+					err = viper.WriteConfig()
+					if err != nil {
+						return err
+					}
+				}
+			}
+			if err = viper.ReadInConfig(); err != nil {
+				if _, ok := err.(viper.ConfigFileNotFoundError); ok {
+					logger.GetLogger().Fatal().Err(err).Msg("Config file not found")
```
Comment (hanahmily) [inline bydbctl/internal/cmd/root.go:49]: You should create a config file if not found.

Comment (sacloudy) [reply bydbctl/internal/cmd/root.go:50]: I think I’ve created it in viper.SafeWriteConfig

**Comment by hanahmily** [comment]
Most cases failed to expect the first one. This might relate to the following place:

https://github.com/apache/skywalking-banyandb/blob/main/banyand/liaison/http/server.go#L125

The HTTP server blocks and the `GracefulStop` doesn't contain any operation to stop the server. Would you please use `http.Server` instead of `http.ListenAndServer` ?

File: bydbctl/internal/cmd/stream.go [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+	StreamCreateCmd := &cobra.Command{
+		Use:     "create",
+		Version: version.Build(),
+		Short:   "banyandb stream schema create Operation",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) (err error) {
+			return cmd.Parent().PersistentPreRunE(cmd.Parent(), args)
+		},
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			client := resty.New()
+			addr, err := cmd.Flags().GetString("addr")
```
Comment (hanahmily) [inline bydbctl/internal/cmd/stream.go:51]: Please use `Addr` and `JSON` as https://github.com/spf13/cobra/blob/main/user_guide.md#working-with-flags

File: bydbctl/internal/cmd/stream.go [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+			if !ok {
+				return errors.New("input json format error")
+			}
+			metadata, ok := stream["metadata"].(map[string]interface{})
+			if !ok {
+				return errors.New("input json format error")
+			}
+			_, ok = metadata["group"].(string)
+			if !ok {
+				metadata["group"] = fmt.Sprintf("%v", viper.Get("group"))
```
Comment (hanahmily) [inline bydbctl/internal/cmd/stream.go:74]: Could you return an error in case the group is absent?

File: bydbctl/internal/cmd/root.go [hunk 4]
```diff
@@ -30,7 +33,41 @@ func NewRoot() *cobra.Command {
...
+					logger.GetLogger().Fatal().Err(err).Msg("Config file was found but another error was produced")
+				}
+				return err
+			}
+			return nil
+		},
 	}
-	// cmd.AddCommand(...)
+	cmd.AddCommand(newBanyanDBCmd()...)
+	Addr := ""
```
Comment (hanahmily) [inline bydbctl/internal/cmd/root.go:68]: You should define Addr and JSON outside to share them with other commands

**Review by hanahmily — CHANGES_REQUESTED** [review]
Please polish your flags defination

File: bydbctl/internal/cmd/stream_test.go [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+		out := capturer.CaptureStdout(func() {
+			err := rootCmd.Execute()
+			Expect(err).NotTo(HaveOccurred())
+		})
+		GinkgoWriter.Write([]byte(out))
+		resp := new(database_v1.StreamRegistryServiceGetResponse)
+		Expect(yaml.Unmarshal([]byte(out), resp)).To(Succeed())
+		GinkgoWriter.Write([]byte(resp.String()))
+		Expect(resp.Stream.Metadata.Name).To(Equal("name1"))
+		Expect(resp.Stream.Metadata.Group).To(Equal("group1"))
```
Comment (hanahmily) [inline bydbctl/internal/cmd/stream_test.go:91]: Please follow them to verify the response

File: bydbctl/internal/cmd/stream.go [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+		Version: version.Build(),
+		Short:   "banyandb stream schema related Operation",
+	}
+
+	StreamCreateCmd := &cobra.Command{
+		Use:     "create",
+		Version: version.Build(),
+		Short:   "banyandb stream schema create Operation",
+		RunE: func(_ *cobra.Command, _ []string) error {
+			return rest("stream", func(request *resty.Request) (*resty.Response, error) {
```
Comment (hanahmily) [inline bydbctl/internal/cmd/stream.go:44]: Use `rest` function to tweak other commands

**Review by hanahmily — CHANGES_REQUESTED** [review]
Tweak your implement based on my examples.

**Review by hanahmily — APPROVED** [review]
LGTM

**Pull request merged**